### DISCUSSION TITLE
Disable CodeglancePro in distraction free mode and zen mode

### DISCUSSION THREAD

**Discussion by saisasidhar** [Nasller/CodeGlancePro#61] [Ideas]
Or atleast a setting that one could use to enable or disable this feature.

**Comment by saisasidhar** [comment]
There is `Ctrl+Shift+G` to disable minimap but it works on one open file at a time. Disabling the plugin is a terrible workaround as it requires a restart.

So I think, an option to disable codeglancepro for all files when entering distractionfree/zen mode and enabling when exiting these modes has added benefits

**Reply by Nasller** [reply]
disabled by default can use?

**Reply by saisasidhar** [reply]
sure. Disabling by default and then re-enabling when exiting sounds apt for "distraction free/zen modes"

**Reply by Nasller** [reply]
i think can make a action to invoke the option

**Reply by saisasidhar** [reply]
Great. Thank you !
